### ISSUE TITLE
Command line fetch returns empty file

### ISSUE THREAD

**Issue by nishraptor** [osfclient/osfclient#159]
Using the command:

osf -u <username> -p xum75 clone <path>

I am able to put in my password and retrieve the file, However, the file turns out to be empty when I load it, when it contains data that can be seen on the osf website. I am also retrieving an error that says 

'osf clone: error: You are not authorized to access this project.' - but I am listed as an administrator on the project.

**Comment by benlindsay** [comment]
Out of curiosity to see if this is related to #144, is this a private or public project?

**Comment by benlindsay** [comment]
I just checked and it looks like it's a private project. I'm guessing whatever the bug is it's related to #144, but let's leave both issues open for now since I'm not positive.

**Comment by nishraptor** [comment]
It is a private project. I saw #144, however, I thought it was a little different since I am able to retrieve the file, except that it is empty. If it helps, it is a .xlsx file.

**Comment by benlindsay** [comment]
good point, I'm guessing whatever bug affected #144 affects this one, but there could be some some differences. There might have been an empty file created that I just missed before, I'll have to try replicating my problem and see.

**Comment by benlindsay** [comment]
@BRosenblatt none on this issue either yet. Are you seeing similar errors?
